pixelpact/src/api.js

```diff
@@ -46,8 +48,9 @@ export function buildFastify(renderFn, compareFn) {
       let actualHtml = request.body.actualHtml;
       let expected = Buffer.from(request.body.expected, "base64");
       let viewport = request.body.viewport;
+      let url = request.body.url ? request.body.url : "/";
```

Comment:
```suggestion
      const` url = request.body.url ?? '/';
```

Reply:
ha, did not know this nullish coalescing operator. Nice!